File: 2-Ad-Hoc/ex2417.c
#include <stdio.h>

int main(){
    //variaveis
    int vitCor, empCor, saldCor, vitFla, empFla, saldFla;
    int totCor, totFla;
    char result; 

    //entrada
    scanf("%d %d %d %d %d %d", &vitCor, &empCor, &saldCor, &vitFla, &empFla, &saldFla);

    //processamento
    totCor = vitCor * 3 + empCor;
    totFla = vitFla * 3 + empFla;

    if(totCor == totFla && saldCor == saldFla){
        result = '=';
    }
    else{
        if(totCor > totFla){
            result = 'C';
        }
        else if(totFla > totCor){
            result = 'F';
        }
        else{
            if(saldCor > saldFla){
                result = 'C';
            }
            else{
                result = 'F';
            }
        }
    }

    //saida
    printf("%c\n", result);

    return 0;
}

File: 2-Ad-Hoc/ex2417.py
#entrada
vitCor, empCor, saldCor, vitFla, empFla, saldFla = str(input()).split()
vitFla = int(vitFla)
empFla = int(empFla)
saldFla = int(saldFla)
vitCor = int(vitCor)
empCor = int(empCor)
saldCor = int(saldCor)

#processamento & saida
totalFla = vitFla * 3 + empFla
totalCor = vitCor * 3 + empCor

if totalFla == totalCor:
    if saldFla > saldCor:
        print('F')
    elif saldCor > saldFla:
        print('C')
    else:
        print('=')
else:
    if totalFla > totalCor:
        print('F')
    else:
        print('C')
